src/many-migration/src/lib.rs

```diff
@@ -362,117 +164,396 @@ impl<T, E> InnerMigration<'static, T, E> {
             description,
         }
     }
-}
 
-impl<'a, T, E> InnerMigration<'a, T, E> {
-    pub const fn name(&self) -> &'a str {
+    #[inline]
+    pub const fn name(&self) -> &str {
         self.name
     }
 
-    pub const fn description(&self) -> &'a str {
+    #[inline]
+    pub const fn description(&self) -> &str {
         self.description
     }
 
+    #[inline]
     pub const fn r#type(&self) -> &'_ MigrationType<T, E> {
         &self.r#type
     }
 
     /// This function gets executed when the storage block height == the migration block height
-    pub fn initialize(&self, storage: &mut T) -> Result<(), E> {
+    fn initialize(&self, storage: &mut T) -> Result<(), E> {
         match &self.r#type {
             MigrationType::Regular(migration) => (migration.initialize_fn)(storage),
+            MigrationType::Hotfix(_) => Ok(()),
             _ => {
-                tracing::trace!(
-                    "Migration {} is not of type `Regular`, skipping",
-                    self.name()
-                );
+                trace!("Skipping migration {}", self.name());
```

Comment:
Modify this message to be consistent with the others.

Reply:
Actually the others are wrong, fixed.